src/screens/EthToSub.tsx

```diff
@@ -200,6 +206,35 @@ const EthToSub = ({ className, ethProvider } : Props) => {
 	);
 };
 
+function parseAmount(val: string|null) {
```

Comment:
Nit: These 2 functions would probably be better placed in the `./util` folder as they can be reused and aren't specific to this component.

Reply:
Done!